si/orange3-text.jaml

```diff
@@ -0,0 +1,5182 @@
+}}
+hr {{
+    border: none;
+    height: 1px;
+    background-color: #000;
+
+}}
+
+</style>
+</head>
+<body>
+{}
+</body>
+</html>
+": false
+    def `run`:
+        Calculating...: Računanje...
+    class `DocumentsModel`:
+        def `_argsortData`:
+            mergesort: false
+    class `DisplayDocument`:
+        Document: Dokument
+        Section: Razdelek
+        Sentence: Stavek
+        <mark data-markjs='true'>: false
+        </mark>: false
+        {START_TAG}{{}}{END_TAG}: false
+        \n: false
+        ...: false
+        def `__call__`:
+            {self.SECTION_SEP}+: false
+            ' ': false
+        def `_replace_sentences`:
+            def `replace_unmatched`:
+                ' ': false
+    class `OWSemanticViewer`:
+        Semantic Viewer: Semantični pregledovalnik
+        Infers characteristic words from the input corpus.: Iz vhodnega korpusa izpelje karakteristične besede.
+        icons/SemanticViewer.svg: false
+        semantic viewer, search: semantic viewer, search, iskanje, pregledovalnik, semantika
+        class `Inputs`:
+            Corpus: Korpus
+            Words: Besede
+        class `Outputs`:
+            Matching Docs: Ujemajoči dokumenti
+            Other Docs: Ostali dokumenti
+            Corpus: Korpus
+        class `Warning`:
+            Input is missing 'Words' column.: V vnosu manjka stolpec 'Besede'.
+        def `_setup_gui`:
+            Filtering: Filtriranje
+            threshold: false
+            'Threshold: ': 'Prag: '
+            Display: Prikaži
+            display_index: false
+        def `set_words`:
+            type: false
+            words: false
+        def `_list_documents`:
+            Match: Ujemanje
+            Score: Rezultat
+            Document: Dokument
+            def `get_avg_score`:
+                NA: /
+        def `_show_documents`:
+            \n: false
+            <br/>: false
+            <p>{text}</p>: false
+            <hr>: false
+        def `send_report`:
+            Corpus: Korpus
+            Words: Besede
+            ', ': false
+            {:.3f}: false
+    __main__: false
+    human: false
+    graph: false
+    minors: false
+    trees: false
+    deerwester: false
+widgets/owsentimentanalysis.py:
+    class `OWSentimentAnalysis`:
+        Sentiment Analysis: Analiza sentimenta
+        Compute sentiment from text.: Izračun sentimenta (čustev) iz besedila.
+        icons/SentimentAnalysis.svg: false
+        sentiment analysis, emotion: sentiment analysis, emotion, čustva, občutki, razpoloženje
+        class `Inputs`:
+            Corpus: Korpus
+        class `Outputs`:
+            Corpus: Korpus
+        class `Warning`:
+            Only one dictionary loaded.: Naložen je samo en slovar.
+            No dictionaries loaded.: Slovarji niso naloženi.
+        class `Error`:
+            'Sentiment cannot be computed since you are offline and the ': '"Sentimenta ni mogoče izračunati, ker ste brez povezave in '
```

Comment:
```suggestion
            'Sentiment cannot be computed since you are offline and the ': 'Sentimenta ni mogoče izračunati, ker ste brez povezave in '
```
Quote probably redundant